doc/downgrade.8.md

```diff
@@ -53,10 +53,9 @@ The columns have the following meaning:
   The release of the package in cache or ALA.
 
 *location*
-  Possible values: {(remote)|(local)}
+  Possible values: {remote|/path/to/cache/dir}
 
-  If you have already downloaded this version, it will show *local*.\
-  *remote* indicates that the version is available on the ALA.
+  If you have already downloaded this version, it will show the cache directory where the package is located. *remote* indicates that the version is available on the ALA.
```

Comment:
Can you maintain the hard-wrapping at 80 for any .md files? I thought Restyled was set up here, and `prettier-markdown` should be doing that automatically :thinking:

Reply:
I can't recall exactly why I did not add all markdowns to `.restyled.yaml`. Will do it now and then take it from there.

Reply:
Edited the markdowns and the `.restyled.yaml` file. For some reason the web-hook did not get triggered with a new commit. Is it possible for you to manually trigger a check?

Reply:
I wouldn't worry about it, could be any number of reasons for the hiccup.